chain/network/src/routing/routing_table_actor.rs

```diff
@@ -44,18 +45,20 @@ pub enum Prune {
 ///   - store removed edges to disk
 ///   - we currently don't store active edges to disk
 pub struct RoutingTableActor {
-    /// Data structure with all edges. It's guaranteed that `peer.0` < `peer.1`.
-    pub edges_info: HashMap<(PeerId, PeerId), Edge>,
+    /// Collection of edges representing P2P network.
+    /// It's indexed by `Edge::key()` key and can be search through by called `get()` function
+    /// with `(PeerId, PeerId)` as argument.
+    pub(crate) edges_info: ProjectedHashMap<(PeerId, PeerId), Edge>,
     /// Data structure used for exchanging routing tables.
     #[cfg(feature = "protocol_feature_routing_exchange_algorithm")]
-    pub peer_ibf_set: crate::routing::ibf_peer_set::IbfPeerSet,
+    pub(crate) peer_ibf_set: crate::routing::ibf_peer_set::IbfPeerSet,
     /// Current view of the network represented by undirected graph.
     /// Nodes are Peers and edges are active connections.
-    pub raw_graph: Graph,
+    pub(crate) raw_graph: Graph,
     /// Active PeerId that are part of the shortest path to each PeerId.
-    pub peer_forwarding: Arc<HashMap<PeerId, Vec<PeerId>>>,
+    pub(crate) peer_forwarding: Arc<HashMap<PeerId, Vec<PeerId>>>,
     /// Last time a peer was reachable through active edges.
-    pub peer_last_time_reachable: HashMap<PeerId, Instant>,
+    pub(crate) peer_last_time_reachable: HashMap<PeerId, Instant>,
```

Comment:
:+1: love this `pub(crate)`'s, they make this immediately clear that the thing isn't accessed outside of this compilation unit.